core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -93,7 +95,8 @@ object WorkspaceService {
                   googleIamDao: GoogleIamDAO,
                   terraBillingProjectOwnerRole: String,
                   terraWorkspaceCanComputeRole: String,
-                  terraWorkspaceNextflowRole: String
+                  terraWorkspaceNextflowRole: String,
+                  snapshotServiceConstructor: RawlsRequestContext => SnapshotService
   )(
```

Comment:
Needed to add this `SnapshotService` into the `WorkspaceService` which led to a lot of changes throughout the code.